salt/netapi/rest_tornado/saltnado.py

```diff
@@ -1113,6 +1113,19 @@ def _disbatch_local_async(self, chunk):
         f_call = self._format_call_run_job_async(chunk)
         # fire a job off
         pub_data = yield self.saltclients['local_async'](*f_call.get('args', ()), **f_call.get('kwargs', {}))
+        if 'jid' not in pub_data:
+            # TODO uncomment after deprecated
+            # please check test case for this tests.integration.netapi.rest_tornado.test_app:test_simple_local_async_post_no_tgt
+            # raise tornado.gen.Return('No minions matched the target. No command was sent, no jid was assigned.')
+
+            import salt.utils.versions
+            salt.utils.versions.warn_until(
```

Comment:
With our new process, we're making it a flat year for deprecations.

Could you change this to `warn_until_date('20210101', ...`?

Reply:
sorry for late response.
changed warning with date.